unity/CAVE/Assets/Resources/Prefabs.meta

```diff
@@ -0,0 +1,12 @@
+fileFormatVersion: 2
+<<<<<<<< HEAD:unity/CAVE/Assets/Resources/Prefabs.meta
+guid: ac120ba708de7634c928b7b058bec684
+========
+guid: cb6dcedd95273ca46beb6e9df64986f8
+>>>>>>>> main:unity/CAVE/Assets/XR Interaction Toolkit/Presets.meta
```

Comment:
This looks like a merge conflict?

Reply:
Interesting... the meta files are written by Unity so I'm curious why it never got overwritten